docs/porting-guide.rst

```diff
@@ -1,4 +1,4 @@
-Porting guide
+Porting Guide
```

Comment:
If you want to change this, you should change it everywhere in the documentation. The other sections do not capitalize after the first one. This document is very long. Maybe a subsequent PR could split it.

Reply:
Since you corrected section header `Reference Counting` to `Reference counting`, I suggest to do [Chicago Manual of Style](https://www.chicagomanualofstyle.org/book/ed17/frontmatter/toc.html) ([here](https://capitalizemytitle.com/style/Chicago/#capitalizationrules) is a quick summary of it) for page captions.

Reply:
Whatever we do we should be consistent. It doesn't really matter which style we use.